documentation-site/components/layout.js

```diff
@@ -91,6 +132,25 @@ class Layout extends React.Component<PropsT, {sidebarOpen: boolean}> {
             role="main"
             $isSidebarOpen={sidebarOpen}
           >
+            {isGitHubEditDisabled ? null : (
+              <Block
+                display={['none', 'block']}
+                position="absolute"
+                right="0px"
+                top="-10px"
+              >
+                <Button
+                  startEnhancer={() => <GithubLogo size={16} color="#666666" />}
```

Comment:
nit: could we use a pencil icon for this button? the github logo for a sec made me think it would take me to the source code